src/main/kotlin/no/nav/syfo/application/BlockingApplicationRunner.kt

```diff
@@ -155,6 +156,21 @@ class BlockingApplicationRunner {
                             loggingMeta)
                     val samhandlerPraksis = samhandlerPraksisMatch?.samhandlerPraksis
 
+                    if (samhandlerPraksisMatch == null &&
+                            samhandlerInfo.firstOrNull()?.samh_praksis != null &&
```

Comment:
Tror kanskje denne null-sjekken er overflødig, men better safe than sorry 😅

Reply:
Jepp tror den er det, men her skal vi ikkje ha noen nullpointere😅

Reply:
1. samhandlerInfo.firstOrNull()?.samh_praksis?.firstOrNull() != null) {
                            val firstSamhnalderPraksis = samhandlerInfo.firstOrNull()?.samh_praksis?.firstOrNull()
2.                        if (firstSamhnalderPraksis != null) {

disse to er vel egentlig de samme?

Reply:
Kan være lurt å skrive enhetstester som tester sånne ting kanskje?